### PR TITLE
Dev Portal config landing pages cleanup

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/DOCS-1052

Direct review link:

This page however needs more adjustment. The dev portal landing pages need revisiting.

https://deploy-preview-2072--kongdocs.netlify.app/enterprise/1.5.x/developer-portal/configuration/

https://deploy-preview-2072--kongdocs.netlify.app/enterprise/1.5.x/developer-portal/configuration/authentication/

### CODE REVIEW

**Comment by tlblessing** [comment]
@lena-larionova this page needs some more rearranging. Please advise. You had a comment about it needing some help earlier this week https://kongstrong.slack.com/archives/G012HLJ4GJJ/p1589847079008300.

File: app/enterprise/1.5.x/developer-portal/configuration/index.md [hunk 1]
```diff
@@ -21,13 +21,7 @@ title: Dev Portal Configuration
     <a href="/enterprise/{{page.kong_version}}/developer-portal/configuration/authentication/oidc">OIDC</a></h3>
     <a href="/enterprise/{{page.kong_version}}/developer-portal/configuration/authentication/oidc">
     Secure the Dev Portal with the OpenID Connect Authentication Plugin &rarr;</a>
-  </div>
```
Comment (tlblessing) [inline app/enterprise/1.5.x/developer-portal/configuration/index.md:24]: added back in 2nd commit

File: app/enterprise/1.5.x/developer-portal/configuration/index.md [hunk 2]
```diff
@@ -5,58 +5,48 @@ title: Dev Portal Configuration
 <div class="docs-grid">
   <h2>Authentication</h2>
```
Comment (lena-larionova) [inline app/enterprise/1.5.x/developer-portal/configuration/index.md:6]: This title doesn't need to be here at all, it only adds confusion.

Comment (tlblessing) [reply app/enterprise/1.5.x/developer-portal/configuration/index.md:6]: > This title doesn't need to be here at all, it only adds confusion.

Well I think it's probably fine here given the rest of the structure but adding custom fields should move.

image

Comment (tlblessing) [reply app/enterprise/1.5.x/developer-portal/configuration/index.md:6]: Discussed with @GayleNeumann; holding off on further changes for now due to forthcoming doc site navigation enhancements.

**Comment by tlblessing** [comment]
Closing per @lena-larionova; will clean up in future release

**Pull request closed without merging**